ga4gh/frontend.py

```diff
@@ -9,20 +9,30 @@
 
 import os
 import datetime
+import functools
 
 import flask
 import flask.ext.cors as cors
 import humanize
+import yaml
+from flask import session, request
 
 import ga4gh
 import ga4gh.backend as backend
 import ga4gh.protocol as protocol
 import ga4gh.exceptions as exceptions
 
+import oic
```

Comment:
The imports are not as clear here, since oic is a complexly layered package. See `STYLE.md` for a quick discussion of the ideal import structure. This is an ideal though, so if oic doesn't really fit this pattern, then whatever works is good.